cmd/createorupdate/createorupdate.go

```diff
@@ -289,6 +293,9 @@ func main() {
 		s := fakerp.NewServer(log, conf.ResourceGroup, fakeRpAddr, conf)
 		go s.ListenAndServe()
 	}
+	if *isAdmin {
+		rpURL += "/admin"
```

Comment:
Instead of doing this we should probably create a client in pkg/admin/api and use that here instead of using `v20180930preview.NewOpenShiftManagedClustersClientWithBaseURI` below.